### DISCUSSION TITLE
怎么导入

### DISCUSSION THREAD

**Discussion by shundebo** [MMmmmoko/CatTuber_Cats#4] [Q&A] [answered]
j

**Comment by MMmmmoko** [comment, marked as answer]
https://github.com/MMmmmoko/CatTuber_Cats/blob/main/docs/3_%E8%87%AA%E5%AE%9A%E4%B9%89%E8%A7%92%E8%89%B2%E6%A8%A1%E5%9E%8B.md   请直接参考这里的文末。  如果图片加载不出来可以使用科学上网工具，或者将此存储库下载后寻找放置于库中的[imgs/img3_2.png](https://github.com/MMmmmoko/CatTuber_Cats/blob/main/docs/imgs/img3_2.png)
